src/DynamoCore/BuiltInAndOperators/BuiltIn.xml

```diff
@@ -135,8 +135,9 @@
     <member name="List.GroupByFunction">
       <summary>Use a function to determine how list items should be grouped.</summary>
       <param name="list">list of values</param>
-      <param name="function"></param>
-      <search>group,function</search>
+      <param name="groupFunction">Function to group list</param>
+      <returns name="groupedList">Grouped list</returns>
+      <search>group,function,groupbyfunction</search>
```

Comment:
I don't think the name of the node needs to be repeated in the search terms as the node name is used in search by default.

Reply:
Also have you verified that it is this XML file that ends up getting copied to the output, en-US folder?

Reply:
> I don't think the name of the node needs to be repeated in the search terms as the node name is used in search by default.

addressed in f0591d0

Reply:
> Also have you verified that it is this XML file that ends up getting copied to the output, en-US folder?

Ok, just verified that this XML file is getting copied to the en-US folder
![image](https://user-images.githubusercontent.com/25138291/93866804-14bbe500-fcc0-11ea-949c-6098af216e0d.png)